calculation/calculation_rule.py

```diff
@@ -171,7 +179,7 @@ def ready(cls):
                 cls.signal_calculate_event.connect(cls.run_calculation_rules, dispatch_uid="on_calculate_event_signal")
 
     @classmethod
-    def active_for_object(cls, instance, context):
+    def active_for_object(cls, instance, context, type='account_receivable', sub_type='contribution'):
```

Comment:
we should have a check it regarding the type/subtype

Reply:
ok. I will look into this in next PR.